README.md

```diff
@@ -359,28 +359,33 @@ Email [help@rainforestqa.com](mailto:help@rainforestqa.com) if you're having tro
 
 ## Release process
 
-Check the `.circleci/config.yml` for the latest, but currently merging to master will build and deploy to the following Equinox channels:
+Check the [`.circleci/config.yml`](.circleci/config.yml) for the latest, but currently merging to master will build and deploy to the following Equinox channels:
 
-Tag                             | Channels
+Tag | Channels
 --------------------------------|-------------
-No tag                          | dev
-vX.Y.Z-alpha.N or vX.Y.Z-beta.N | beta, dev
-vX.Y.Z                          | stable, beta, dev
+No tag | `dev`
+`vX.Y.Z-alpha.N` or `vX.Y.Z-beta.N` | `beta`, `dev`
+`vX.Y.Z` | `stable`, `beta`, `dev`
 
-Development + release process is:
+### Development + release process is:
 
+#### Development PR
 1. Branch from master
-2. Do work
-3. Open PR against master
-4. Merge to master
-5. Branch from master to update `CHANGELOG.md` to include the commit hashes and release date
-6. Update the `version` constant in `rainforest-cli.go` following [semvar](http://semver.org/)
-7. Merge to master
-8. Tag the master branch with the release:
+1. Do work
+1. Open PR against master
+1. Merge to master
+#### Changelog PR
+1. Branch from master to update `CHANGELOG.md` to include the commit hashes and release date
+1. Update the `version` constant in `rainforest-cli.go` following [semvar](http://semver.org/)
```

Comment:
Typo with "semvar" but might be better to just write it all out so it's more clear
```suggestion
1. Update the `version` constant in `rainforest-cli.go` following [semantic versioning](http://semver.org/)
```